src/modules/list.js

```diff
@@ -1,118 +1,131 @@
-/* eslint-disable  no-use-before-define */
-
-let tasks = [];
-
-tasks.sort((a, b) => a.index - b.index);
-
-function saveTasksToLocalStorage() {
-  localStorage.setItem('tasks', JSON.stringify(tasks));
-}
-
-function updateTaskIndexes() {
-  tasks.forEach((task, index) => {
-    task.index = index;
-  });
-}
-
-export function createItem() {
-  const list = document.getElementById('list');
-  list.innerHTML = '';
-  tasks.forEach((task) => {
-    const aTask = document.createElement('li');
-    list.appendChild(aTask);
-    aTask.classList.add('list-li');
+export default class List {
+  constructor() {
+    this.list = JSON.parse(localStorage.getItem('todo-list'));
+    if (!this.list) {
+      this.list = [];
+    }
+    this.display();
+  }
```

Comment:
You have done a great work by implementing this class 🥇  and setting the list from localStorage in the constructor is a good idea 💯